### PR TITLE
Upgrading accepts to fix jaredallard/nexe#219

### PR DESCRIPTION
**Pull request by palavrov**

### CODE REVIEW

**Comment by medemi68** [comment]
Hopefully this is accepted soon.

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "debug": "2.2.0",
     "ws": "1.0.1",
     "engine.io-parser": "1.2.4",
-    "accepts": "1.1.4"
+    "accepts": "^1.3.3"
```
Comment (nkzawa) [inline package.json]: please don't use `^`, otherwise LGTM

Comment (palavrov) [inline package.json]: Fixed!

It was added by npm.

**Comment by john0312** [comment]
This seemed to be fixed by by commit 298cb6fd6f1e6ab63c2a3fb5f5d762643c138135. However, that commit is not included in v1.6.11, but is instead present in the current master branch, because 1.6.10-pre is merged. Could @rauchg release another version base on the current master branch (966344219d3c20439a75ee0323c86a148922088c) so that jaredallard/nexe#219 can be fixed? Thank you.

**Comment by palavrov** [comment]
@john0312, this PR fixes another regression with 6d92897

**Pull request closed without merging**